src/redux/categories/categories.js

```diff
@@ -0,0 +1,25 @@
+const UNDER_CONSTRUCTION = 'UNDER_CONSTRACTION';
+const COMPELET = 'COMPLETE';
```

Comment:
- [ ] Kindly ensure that the value of this action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), which should look like:      
```javascript
 const COMPLETE = 'bookstore/categories/COMPLETE';